ext/ffi/00_ffi.js

```diff
@@ -6,7 +6,52 @@
   const __bootstrap = window.__bootstrap;
   const {
     ArrayBuffer,
+    BigInt,
+    Number,
+    TypeError,
   } = window.__bootstrap.primordials;
+
+  function unpackPointer([hi, lo]) {
+    return BigInt(hi) << 32n | BigInt(lo);
+  }
+
+  function packPointer(value) {
+    return [Number(value >> 32n), Number(value & 0xFFFFFFFFn)];
+  }
+
+  class UnsafePointer {
+    value;
```

Comment:
Could `valueOf` method be implemented as well?

Reply:
Hmm, yeah but im undecided if we want pointer values to be immutable because the `UnsafePointer` constructor allows the user to create new pointers from an arbitary value so why not then allow the value property to be mutable and just leave the value property exposed.

Reply:
Ah, no I didn't mean making value immutable, but just adding `valueOf` method too, it would allow things such as `new UnsafePointer(1n) + 2n`

Reply:
ah ok, didn't know about that but sure!